Doc and config (#107)
elastic/apm-agent-ios: docs/setup.asciidoc

<1> APM Server URL
<2> Set secret token for APM server connection

> [[View Instrumentation]]

Reviewer: ```suggestion
[discrete]
[[view-instrumentation]]
=== View instrumentation
```

> The Agent provides SwiftUI.View & UIViewController instrumentation, where the load time of a View is measured using spans.
> All Views simultaneously loaded will be grouped under the same starting span.
> The spans' names will be dictated by the following rules from least to highest precedence:

Reviewer: ```suggestion
The agent provides SwiftUI.View and UIViewController instrumentation, where the load time of a View is measured using spans.
All Views simultaneously loaded will be grouped under the same starting span.
The spans' names will be dictated by the following rules, from least to highest precedence:
```

`AllProductsList.swift`
[source,swift,linenums, highlight=12]
----
struct AllProductsList: View {
    @EnvironmentObject var modelData : ModelData

    var body: some View {
        VStack {
            List(modelData.products, id: \.id) { product in
                AdminProductRow(product: product)

            }
        }.onAppear  {
            modelData.loadProducts()
        }.navigationTitle("All Products")
    }
}
----

You'll see "All Products - view appearing" in Kibana.

If it isn't possible to set a navigation title, use `reportName(_ name: String) -> View` to set the name that will show in Kibana:

`AllProductsList.swift`
[source,swift,linenums, highlight=12]
----
struct AllProductsList: View {
    @EnvironmentObject var modelData : ModelData

    var body: some View {
        VStack {
            List(modelData.products, id: \.id) { product in
                AdminProductRow(product: product)

            }
        }.onAppear  {
            modelData.loadProducts()
        }.reportName("All Products - view appearing")
    }
}
----

> Note: The whole string `All Products - view appearing` needs to be inserted to match the default formatting used for the other two naming options.

Reviewer: ```suggestion

NOTE: The entire string `All Products - view appearing` must be inserted to match the default formatting used for the other two naming options.
```
